fl4health/reporting/wandb_reporter.py

```diff
@@ -110,6 +110,13 @@ def start_run(self, wandb_init_kwargs: dict[str, Any]) -> None:
             id=self.id,
             **wandb_init_kwargs  # Other less commonly used kwargs
         )
+
+        if isinstance(wandb_run, wandb.wandb_run.Run):
```

Comment:
Was this causing an issue somewhere that you saw? Just curious if this is somehow being missed by our mypy or something else.

Reply:
It was mypy that complained about this on my end. I assumed we want to make sure that the initialization returns a `Run` type hence the change here. Is this change okay?

Reply:
Yes for sure. I'm just not sure why I'm not getting that popping up in mypy on my end. I think we recently upgraded mypy versions. Maybe just quickly check that you have the most recent one in your environment and if it fixes the mypy compliant by upgrading if you don't? Otherwise, I'm good to keep this around.

Reply:
Nice. Mystery solved 🙂